### PR TITLE
Use tools.go to manage tools dependency

### PR DESCRIPTION
**Pull request by YangKeao**
### What problem does this PR solve? 

`go get` controller-gen everytime to upgrade it in time.

### CODE REVIEW

File: tools.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// +build tools
+
+package tools
+
+import (
+	_ "sigs.k8s.io/controller-tools/cmd/controller-gen"
```
[the remark this answers is not in the document]
Comment (YangKeao) [reply tools.go]: Can I ignore this?!

[the remark this answers is not in the document]
Comment (fewdan) [reply tools.go]: > Can I ignore this?!

I think so... Or you can add a comment as it said?

File: tools.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// +build tools
```
Comment (cwen0) [inline tools.go]: LICENSE?

**Pull request merged**